### PR TITLE
port makes get_hearers_in_view() faster

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

tgstation/tgstation#60093
tgstation/tgstation#60219
tgstation/tgstation#50269

## Why It's Good For The Game

sanic

## Changelog
:cl:
code: Optimized radio messages
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
It's merged

**Comment by jupyterkat** [comment]
~extools failing. investigate later~ nevermind i'm retarded

File: code/__HELPERS/game.dm [hunk 1]
```diff
@@ -269,48 +255,31 @@
...
+	for(var/atom/movable/movable in view(view_radius, center_turf))
+		var/list/recursive_contents = LAZYACCESS(movable.important_recursive_contents, RECURSIVE_CONTENTS_HEARING_SENSITIVE)
+		if(recursive_contents)
+			. += recursive_contents
+	center_turf.luminosity = lum
 
 /proc/get_mobs_in_radio_ranges(list/obj/item/radio/radios)
 	. = list()
 	// Returns a list of mobs who can hear any of the radios given in @radios
 	for(var/obj/item/radio/R in radios)
```
Comment (ike709) [inline code/__HELPERS/game.dm:275]: Out of curiosity can this be made typeless?

**Review by ike709 — DISMISSED** [review]

File: code/__HELPERS/game.dm (hunk 1 above)
Comment (jupyterkat) [reply code/__HELPERS/game.dm:275]: not sure if things like lighting overlays and turfs should hear. and the `important_recursive_contents` var is only on atom/movable

**Comment by ike709** [comment]
https://github.com/tgstation/tgstation/pull/60297

**Review by ike709 — APPROVED** [review]

**Comment by ike709** [comment]
YOLO

**Pull request merged**